packages/addons/service/snapclient/source/addon.py

```diff
@@ -11,7 +11,7 @@
 
 card = ''
 cards = []
-lines = subprocess.check_output([SNAPCLIENT, '--list']).splitlines()
+lines = lines = subprocess.getoutput(SNAPCLIENT + ' --list').splitlines()
```

Comment:
Why `lines = lines = ` ?

Also you can send PR to upstream project (LibreELEC).